java-frontend/src/main/java/org/sonar/java/ast/JavaAstScanner.java

```diff
@@ -54,8 +51,8 @@ public JavaAstScanner(ActionParser<Tree> parser, @Nullable SonarComponents sonar
   }
 
   public void scan(Iterable<File> files) {
-    ProgressReport progressReport = new ProgressReport("Report about progress of Java AST analyzer", TimeUnit.SECONDS.toMillis(10));
-    progressReport.start(Lists.newArrayList(files));
+//    ProgressReport progressReport = new ProgressReport("Report about progress of Java AST analyzer", TimeUnit.SECONDS.toMillis(10));
+//    progressReport.start(Lists.newArrayList(files));
 
     boolean successfullyCompleted = false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "successfullyCompleted" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)